turbolinks/build.gradle

```diff
@@ -34,6 +36,7 @@ dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'org.apache.commons:commons-lang3:3.4'
     compile 'com.google.code.gson:gson:2.3.1'
+    compile "com.android.support:appcompat-v7:$supportLibVersion"
```

Comment:
Super nitpick -- I like to keep dependencies alphabetized by package name. And yes, I do realize most of them are already out of order. 😬